ccd-definition/CaseEventToComplexTypes/create-order/create-order-prepare-for-hearing.json

```diff
@@ -0,0 +1,94 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "OrderTypeAndDocumentComplexType",
+    "CaseEventID": "createOrder-PREPARE_FOR_HEARING",
+    "CaseFieldID": "orderTypeAndDocument",
+    "ListElementCode": "orderType",
+    "EventElementLabel": "Type of order",
+    "FieldDisplayOrder": 1,
+    "DisplayContext": "MANDATORY"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "OrderTypeAndDocumentComplexType",
+    "CaseEventID": "createOrder-PREPARE_FOR_HEARING",
+    "CaseFieldID": "orderTypeAndDocument",
+    "ListElementCode": "document",
+    "EventElementLabel": "Order document",
+    "FieldDisplayOrder": 2,
+    "DisplayContext": "READONLY",
+    "FieldShowCondition": "document=\"DO_NOT_SHOW\""
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "GeneratedOrderComplexType",
+    "CaseEventID": "createOrder-PREPARE_FOR_HEARING",
+    "CaseFieldID": "order",
+    "ListElementCode": "orderTitle",
```

Comment:
Still think prepending order is superfluous

Reply:
I agree, I'll refactor - (might inflate PR a little, but you can approve it James 😉 )